libmusic/include/lmtypes.h

```diff
@@ -89,25 +91,34 @@ typedef enum : int32_t {
 
 typedef enum {
     cq_unknown = 0,
-    cq_major,
-    cq_minor,
-    cq_7th,
-    cq_major_7th,
-    cq_minor_7th,
-    cq_6th,
-    cq_minor_6th,
-    cq_diminished,
-    cq_half_diminished_7th,
-    cq_9th,
-    cq_major_9th,
-    cq_added_9th,
-    cq_minor_9th,
-    cq_suspended_4th,
-    cq_suspended_2nd,
-    cq_diminished_7th,
-    cq_5th,
-    cq_Min = cq_major,
-    cq_Max = cq_half_diminished_7th
+    cq_maj,
+    cq_min,
+    cq_5,
+    cq_7,
+    cq_maj7,
+    cq_min7,
+    cq_sus2,
+    cq_sus4,
+    cq_hdim7,
+    cq_aug,
+    cq_dim,
+    cq_dim7,  /* Don't set cq_Max above this value */
```

Comment:
why `cq_Max` is set above this value then?

Reply:
I'll update the comment, looks like formulation is confusing.